examples/js/controls/OrbitControls.js

```diff
@@ -178,7 +178,13 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 			position.copy( scope.target ).add( offset );
 
-			scope.object.lookAt( scope.target );
+			var globalTarget = scope.target.clone();
```

Comment:
Avoid calling `clone()` in a tight loop.

Reply:
Is it better to preallocate globalTarget and use copy?